src/Features/CSharp/Portable/EditAndContinue/SyntaxComparer.cs

```diff
@@ -53,6 +53,8 @@ internal enum Label
             // Top level syntax kinds
             CompilationUnit,
 
+            GlobalStatement,
```

Comment:
I'm confused by this and the comment on the enum that says:

> All descendants of a node whose kind is listed here will be ignored regardless of their labels.

Isn't everything a descendant of CompilationUnit, and GlobalStatement is even a direct child?

Reply:
Yes.. as is `NamespaceDeclaration`, `ExternalAliasDirective` and `UsingDirective`. I'm honestly not sure what that last assumption really means - whether nodes are descended into is controlled by the `isLeaf` output parameter on the `GetLabel` method, not by the enum directly. Perhaps it doesn't apply any more?

@tmat ?

Reply:
I don't remember. Probably stale comment.